eskipfile/remote.go

```diff
@@ -172,13 +182,22 @@ func (client *remoteEskipFile) DownloadRemoteFile() error {
 }
 
 func (client *remoteEskipFile) getRemoteData() (io.ReadCloser, error) {
+
 	resp, err := client.http.Get(client.remotePath)
 	if err != nil {
 		return nil, err
 	}
+
 	if resp.StatusCode != 200 {
+		if resp.StatusCode == 304 {
+			return nil, ErrContentNotChanged
+		}
 		return nil, errors.New("download file failed")
```

Comment:
Maybe add status code in this error

Reply:
Lets add status code to the error via fmt.Errorf(...)